src/components/TransactionFormInput.test.js

```diff
@@ -5,9 +5,12 @@ import renderer from 'react-test-renderer'
 describe('Transaction Input Form', () => {
   it('renders correctly', () => {
     const noOp = () => {}
-    const number = '15'
+    const transactionvalue = '15'
```

Comment:
The value is saved as a string and not as a number. Is that correct?

Reply:
Good point, it should be a number